actors-akka/src/main/scala/reactivemongo/actors/actor/package.scala

```diff
@@ -0,0 +1,16 @@
+package reactivemongo.actors
+
+package object actor {
+  type ActorSystem = akka.actor.ActorSystem
+  val ActorSystem = akka.actor.ActorSystem
+
+  type ActorRef = akka.actor.ActorRef
+  type Actor = akka.actor.Actor
+  val Actor = akka.actor.Actor
+
+  type Props = akka.actor.Props
+  val Props = akka.actor.Props
+  type Terminated = akka.actor.Terminated
+  type Cancellable = akka.actor.Cancellable
+
```

Comment:
```suggestion
```

Reply:
Addressed in 87d3301898f6bbf86f7167a2c141cc2bbd9e8370